src/components/ChatMessage/ChatMessage.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { isMobile } from 'react-device-detect'
+// May use this later for resizable query output modal
```

Comment:
I think we should remove all these comments if there is no immediate plan to enable this feature, as we want to maintain a production level code base